client/dashboard/data/site.ts

```diff
@@ -108,7 +108,7 @@ export interface Site {
 	site_owner: number;
 	jetpack: boolean;
 	jetpack_connection: boolean;
-	jetpack_modules: string[] | null;
+	jetpack_modules?: string[] | null; // Can be undefined query cache is prefilled from old Calypso Redux store.
```

Comment:
I think `jetpack_modules` always exists? We are requesting the field in the old Redux world here

https://github.com/Automattic/wp-calypso/blob/d96d770362175e1f6bab8a8ec9bfbf1175f28c76/client/state/sites/constants.js#L12

Reply:
I stand corrected 😄 2d4e1fb0ff7b1b3d4d69718cc32bd6373c30a22f